src/nls/root/strings.js

```diff
@@ -601,7 +602,8 @@ define(function (require, exports, module) {
                 RECENT_DOC: "Recent Documents",
                 DOCUMENT: "Documents",
                 MENU_COMMAND: "Menu Commands",
-                LAYER: "Layers",
+                ALL_LAYER: "Layers",
+                CURRENT_LAYER: "Layers",
```

Comment:
@iwehrman Would `ALL_LAYER` for all open document layers and just `LAYER` for current document layers be better? The reason they need different category names is so that the search store can keep them separate.